src/ripple/core/Config.h

```diff
@@ -162,6 +162,8 @@ class Config : public BasicConfig
 
     std::string START_LEDGER;
 
+    std::optional<uint256> TRAP_TX_HASH = {};
```

Comment:
You do not need squigly brackets here

Reply:
I like initializers to be explicit, it makes reading easier.

Reply:
Most of the rest of rippled doesn't do this. It's a nit, but I prefer the version without the explicit initialization (for example, the string above isn't initialized like this). Let's not bike shed this too much either - it's not a huge deal one way or the other, and I understand your preference, but let's stick to relying on default constructors for now.